carapace-server/src/main/java/org/carapaceproxy/server/ClientConnectionHandler.java

```diff
@@ -132,6 +136,7 @@ protected void channelRead0(ChannelHandlerContext ctx, Object msg) {
                 currentRequest.clientRequestFinished(trailer);
                 totalRequests.inc();
                 runningRequests.inc();
+                listenerRequests.inc();
```

Comment:
Is there a difference from totalRequests?

Reply:
None, the sum of each listener requests is propertly equal to the total amount.